### ISSUE TITLE
How to get request.contentLength

### ISSUE THREAD

**Issue by daveleenew** [flutterchina/dio#866]
### New Issue Checklist

- [ ] I have searched for a similar issue in the [project](https://github.com/flutterchina/dio/issues) and found none

### Issue Info

| Info                           | Value                                                 |      |
| ------------------------------ | ----------------------------------------------------- | ---- |
| Platform Name                  |  flutter                  |      |
| Platform Version               | v1.17.5                             |      |
| Dio Version                    | 3.0.9                                   |      |
| Android Studio / Xcode Version | e.g. Android Studio 3.3.2 / Xcode 10.2.1              |      |
| Repro rate                     | e.g. all the time (100%) / sometimes x% / only once   |      |
| Repro with our demo prj        | e.g. does it happen with our demo project?            |      |
| Demo project link              | e.g. link to a demo project that highlights the issue |      |

### Issue Description and Steps

How to get request contentLength?
![image](https://user-images.githubusercontent.com/34392697/88044185-8d97a700-cb80-11ea-9c9c-f3997cd074ef.png)

I want to get the request content Length with Dio?

**Comment by tejHackerDEV** [comment]
@daveleenew  I was also doing the same thing did you find anything on how to upload file to azure from dio?
